source/common/res/features/budget-daily-spending/main.js

```diff
@@ -21,11 +21,24 @@
         },
 
         observe(changedNodes) {
-          if (changedNodes.has('budget-inspector')) {
+          if (changedNodes.has('navlink-budget active') ||
+              changedNodes.has('budget-table-cell-available-div user-data') ||
+              changedNodes.has('budget-inspector')) {
             ynabToolKit.dailySpending.invoke();
```

Comment:
I thought that one of the changes you wanted me to do from the #714 PR
Checking for 'budget-inspector' was working just fine, I will revert it back.